index.d.ts

```diff
@@ -1090,22 +1090,23 @@ export interface MenuOptionsGroupProps<T> {
   options: Array<{ value: T, label: string }>
 }
 
-export class Menu extends React.PureComponent<MenuProps> {
+declare const MenuItem: React.FC<MenuItemProps>
+declare const MenuDivider: React.FC<{}>
+declare const MenuGroup: React.FC<MenuGroupProps>
+declare const MenuOption: React.FC<MenuOptionProps>
+declare class MenuOptionsGroup<T> extends React.FC<MenuOptionsGroupProps<T>> {}
```

Comment:
Couldn't figure out a quick way to get static members to respect static members.